lib/instrumentation/express-utils.js

```diff
@@ -28,7 +34,13 @@ function getStackPath (req) {
 
 // This function is also able to extract the path from a Restify request as
 // it's storing the route name on req.route.path as well
-function getPathFromRequest (req, useBase) {
+function getPathFromRequest (req, useBase, usePathAsTransactionName) {
+  // Enable usePathAsTransactionName config
+  if (usePathAsTransactionName) {
```

Comment:
You will also need to check that `parseUrl` exists here.

Reply:
No, it will always exist as we fall back to the core module above

Reply:
Oh, my bad. Misread. 😅